modal/cli/container.py

```diff
@@ -71,7 +79,11 @@ async def exec(
     )
     res: api_pb2.ContainerExecResponse = await client.stub.ContainerExec(req)
 
-    await _ContainerProcess(res.exec_id, client).attach(pty=pty)
+    if pty:
+        await _ContainerProcess(res.exec_id, client).attach()
+    else:
+        # TODO: redirect stderr to its own stream?
```

Comment:
Yeah maybe it should go in its own stream. The API is implemented like this to match `subprocess`, but we could easily add `StreamType.STDERR`.